webui/react/src/e2e/tests/auth.spec.ts

```diff
@@ -62,13 +62,8 @@ test.describe('Authentication', () => {
     await signInPage.goto();
     await expect.soft(signInPage.detAuth.submit.pwLocator).toBeDisabled();
     await signInPage.detAuth.username.pwLocator.fill('chubbs');
-    await expect(async () => {
-      // on CI, machines are slow and each click can take two seconds!
-      await expect(signInPage.detAuth.errors.close.pwLocator).toHaveCount(0, { timeout: 3000 });
-      await signInPage.detAuth.submit.pwLocator.click();
-      await signInPage.detAuth.submit.pwLocator.click();
-      // the errors will dissapear after 3 seconds, so let's timeout with 3
-      await expect(signInPage.detAuth.errors.close.pwLocator).toHaveCount(2, { timeout: 3000 });
-    }).toPass({ timeout: 30000 });
+    await signInPage.detAuth.submit.pwLocator.click();
+    await signInPage.detAuth.submit.pwLocator.click();
+    await expect(signInPage.detAuth.errors.close.pwLocator).toHaveCount(2);
```

Comment:
Why do we keep needing to check 2 errors at once? If we do need to check that it errors twice, can we check them one-after the other rather than trying to check `count(2)`. This seems flaky and too in the weeds for an end to end test. 
Another way to think about it: If the app changed to only show one error at a time(the second error replacing the first) and this failed, would that even be a bug? Both errors still showed so the user isn't missing info, we just wouldn't have 2 toasts at once which seems fine to me. That actually seems how most errors work. I just tried a few sites (linkedin, google, facebook, instagram) and none of them ever show 2 errors. If you click twice, you just see the same error in the same spot.

Reply:
honestly this is more of a test that the error util does what we're expecting it to. it's not directly an auth test, but it's related and i wanted to keep it in. (i.e. make sure we don't replace or discard error messages). i could leave the justification in a comment on the test case.